expr/shaper.go

```diff
@@ -208,6 +205,9 @@ func castRecordType(zctx *zson.Context, input, spec *zng.TypeRecord) (*zng.TypeR
 }
 
 func castType(zctx *zson.Context, inType, specType zng.Type) (zng.Type, error) {
+	if _, ok := specType.(*zng.TypeMap); ok {
+		return nil, fmt.Errorf("cannot yet use maps in shaping functions")
```

Comment:
Is there an issue tracking this? Maybe include the issue in the error?

Reply:
Good idea. I created #2894 and added it to this error string in 0926fce46aed162fe1b807b3c618392b16badc5c.